debian/rules

```diff
@@ -199,8 +199,8 @@ binary-common:
 	dh_installexamples
 	dh_installman
 	dh_link
-	dh_strip -ppython-thrift --dbg=python-thrift-dbg
-	dh_strip -ppython3-thrift --dbg=python3-thrift-dbg
+	dh_strip -ppython-thrift --dbg-package=python-thrift-dbg
+	dh_strip -ppython3-thrift --dbg-package=python3-thrift-dbg
```

Comment:
Actually, -dbg= was from original line from THRIFT-71.
I don't know much about this but seeing the doc it seems this fix seems correct.